catalog/spatial/registry/registry-federation-admin-service-impl/src/main/java/org/codice/ddf/registry/federationadmin/service/impl/RefreshRegistryEntries.java

```diff
@@ -116,66 +116,119 @@ public void refreshRegistryEntries() throws FederationAdminException {
                 remoteResults.getRemoteRegistryMetacards();
         Map<String, Metacard> registryMetacardsMap = getRegistryMetacardsMap();
 
+        if (!remoteResults.getFailureList()
+                .isEmpty()) {
+            LOGGER.debug("Failed to query the following registry stores: {}",
```

Comment:
Wrap this in isDebugEnabled to avoid uneccessary string manipulations, or just remove them and let the toString of the collection do it for you during the logging.